datasets/sem_eval_2020_task_11/sem_eval_2020_task_11.py

```diff
@@ -151,7 +165,7 @@ def _split_generators(self, dl_manager):
             ),
         ]
 
-    def _generate_examples(self, data_dir, keys, split, labels):
+    def _generate_examples(self, data_dir, keys, split, labels, **kwargs):
```

Comment:
```suggestion
    def _generate_examples(self, data_dir, keys, split, labels, tc_test_template):
```
We can directly have the right argument here.

For the train and val set, you can just set `"tc_test_template": None` in their `gen_kwargs`

Reply:
Done!